.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": false
```

Comment:
This should be in the `workspace.code-workspace` file. Although, I wouldn't worry too much about adding quality of life improvements to this repo because the plan is to migrate it to https://github.com/spear-ai/ui.

Reply:
Removed.